src/drawer/components/EmptyScreen/EmptyScreen.tsx

```diff
@@ -15,7 +15,7 @@ import {
 } from './EmptyScreen.style';
 
 interface EmptyScreenProps {
-  type: 'SEARCH' | 'STAR' | 'MYDRAWER' | 'PROVIDER';
+  type: 'SEARCH' | 'STAR' | 'MYDRAWER' | 'CATEGORY';
```

Comment:
`'SEARCH' | 'STAR' | 'MYDRAWER' | 'CATEGORY'` 타입이 `empty.contant.ts` 파일에서도 쓰이는 것 같은데 별도의 타입으로 분리하면 어떨까요?

```suggestion
export type EmptyStateType = 'SEARCH' | 'STAR' | 'MYDRAWER' | 'CATEGORY';
```

Reply:
오 따로 빼면 재사용성하기에도 좋고 깔끔하게 코드 작성할 수 있겠네요 따로 빼둘게요!